recipes-ni/ni-utils/ni-utils_1.0.bb

```diff
@@ -11,7 +11,10 @@ SRC_URI = "file://status_led \
 "
 
 FILES_${PN} += "\
-	/usr/lib/nisetbootmode.functions \
+	${bindir}/status_led \
+	${libdir}/nisetbootmode.functions \
+	${sysconfdir}/init.d/nisetbootmode \
```

Comment:
Again, why move nisetbootmode here? Cleanup?

Reply:
nisetbootmode is part of this package already (see below in do_install). I just added it to FILES section.